stories/setup/webpack.config.babel.js

```diff
@@ -48,10 +50,17 @@ module.exports = {
       /config$/,
       'core/client/config.js',
     ),
+
+    // This swaps the server side window object with a standard browser window.
+    new webpack.NormalModuleReplacementPlugin(
```

Comment:
It looks like this whole plugin array can be built with [`getPlugins()`](https://github.com/mozilla/addons-frontend/blob/master/webpack-common.js#L140) -- did you try that? If it doesn't work, maybe you can add some parameters to that function for how you need it to change?

Reply:
I think I had that originally so let me revisit this one : )

Reply:
the circular plugin causes issue it seems so if I add a check for that I think this will work